spyder/plugins/completion/kite/plugin.py

```diff
@@ -34,16 +34,32 @@ def __init__(self, parent):
         enable_code_snippets = CONF.get('lsp-server', 'code_snippets')
         self.client = KiteClient(None, enable_code_snippets)
         self.kite_process = None
+        self.main_window_visible = False
+        self.kite_initialized = False
+        self.onboarding_shown = False
```

Comment:
Is this attribute needed? or maybe it was a placeholder for the value that we should get from the preferences?

Reply:
sorry, I should have removed this. 
The setting is called `kite_show_onboarding`, is that okay or should I move it to another section or config file?

Reply:
I think is ok :+1: